features/step_definitions/command_steps.rb

```diff
@@ -395,3 +395,13 @@
     end
   end
 end
+
+And (/^I create the "([^"]*)" bootstrap-repo for "([^"]*)" on the server$/) do |arch, target|
+  node = get_target(target)
+  os_version_raw, _code = node.run('grep "VERSION=" /etc/os-release')
+  os_version = os_version_raw.strip.split('=')[1].delete '"'
+  puts 'i should see the os version: ' + os_version
+  command = 'mgr-create-bootstrap-repo -c SLE-' + os_version + '-' + arch
+  puts 'running the command on the server:' + command
+  _out, $fail_code = $server.run(command, false)
```

Comment:
$fail_code is a used for some steps that need to check the fail_code, that"s why we use a global var for passing it to other steps